11_circular_csimeone/scr/plot_gages.R

```diff
@@ -0,0 +1,122 @@
+# Plot gage locations
+
+library(tibble)
+library(dplyr)
+library(stringr)
+library("ggplot2")
+library("cowplot")
```

Comment:
I'd suggest using the same syntax here as the other packages to be consistent

Reply:
Done